### PR TITLE
Included Mountain layout.

### PR DESCRIPTION
**Pull request by Roidujeu**
Made some other small changes.

### CODE REVIEW

File: config.def.h [hunk 1]
```diff
@@ -54,45 +61,75 @@ static const Layout layouts[] = {
 #define SHCMD(cmd) { .v = (const char*[]){ "/bin/sh", "-c", cmd, NULL } }
 
 /* commands */
-static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
-static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_cyan, "-sf", col_gray4, NULL };
+static const char *dmenucmd[] = { "dmenu_run",  NULL };
 static const char *termcmd[]  = { "st", NULL };
+static const char *sLock[]  = { "slock", NULL };
+static const char *backlitInc[]  = { "sudo", "brightnessctl", "set", "+2", NULL };
```
Comment (L04DB4L4NC3R) [inline config.def.h]: Why was this added?

Comment (L04DB4L4NC3R) [reply config.def.h]: No sudo commands should be keybound.

File: config.def.h [hunk 2]
```diff
@@ -54,45 +61,75 @@ static const Layout layouts[] = {
...
+static const char *backlitDec[]  = { "sudo", "brightnessctl", "set", "2-", NULL };
+static const char *volToggle[]  = { "pactl", "set-sink-mute", "0", "toggle", NULL };
+static const char *volDown[]  = { "pactl", "set-sink-volume", "0", "-2%", NULL };
+static const char *volUp[]  = { "pactl", "set-sink-volume", "0", "+1%", NULL };
```
Comment (L04DB4L4NC3R) [inline config.def.h]: Additional dependancies are required. What if the user does not have PulseAudio?

File: config.def.h [hunk 3]
```diff
@@ -54,45 +61,75 @@ static const Layout layouts[] = {
...
+static const char *audToggle[]  = { "cmus-remote", "-u", NULL };
+static const char *audNext[]  = { "cmus-remote", "-n", NULL };
+static const char *audPrev[]  = { "cmus-remote", "-r", NULL };
+static const char *nnnSpawn[] = { "st", "nnn", NULL };
+static const char *sudonnnSpawn[] = { "sudo", "st", "nnn", NULL };
```
Comment (L04DB4L4NC3R) [inline config.def.h]: Sudo user again!

File: config.def.h [hunk 4]
```diff
@@ -54,45 +61,75 @@ static const Layout layouts[] = {
...
+static const char *alsamixer[] = { "st", "alsamixer", NULL };
```
Comment (L04DB4L4NC3R) [inline config.def.h]: Is it the simple terminal? If yes then the users who dont use it will have to make changes everywhere. Have a global TERMINAL definition in the config and inject it in functions like this

File: polygonwm.c [hunk 5]
```diff
@@ -328,18 +332,94 @@ spiral(Monitor *mon) {
...
 
+void dewdrops(Monitor *mon) {
+	unsigned int i, n, nx, ny, nw, nh;
+	Client *c;
+
+	for(n = 0, c = nexttiled(mon->clients); c; c = nexttiled(c->next), n++);
+	if(n == 0) {
+		return;
+	}
+	nx = mon->wx;
```
Comment (L04DB4L4NC3R) [inline polygonwm.c]: Need comments here. No idea what nx and ny mean. Suckless is known for using bad variable name and not commenting enough. Let us not do the same

**Review by L04DB4L4NC3R — CHANGES_REQUESTED** [review]

File: polygonwm.c [hunk 6]
```diff
@@ -2215,6 +2322,58 @@ roundcorners(Client *c)
...
+{
+    Window w = c->win;
+    XWindowAttributes wa;
+    XGetWindowAttributes(dpy, w, &wa);
+
+    if(!XGetWindowAttributes(dpy, w, &wa)) {
+		return;
+	}
+
+    int width = (borderpx * 2) + wa.width;
```
Comment (L04DB4L4NC3R) [inline polygonwm.c:2312]: Where did this `2` come from? Is the boundary configurable?

**Review by L04DB4L4NC3R — CHANGES_REQUESTED** [review]

File: polygonwm.c (hunk 6 above)
Comment (Roidujeu) [reply polygonwm.c:2312]: this is not an issue. 2 is for two sides of the window. borderpx is the no of pixels that should be taken as border and that multiplied by 2 gives the total no of pixels that should be added to the width. And yes, the border width can be specified in the config file.

File: config.def.h (hunk 4 above)
Comment (Roidujeu) [reply config.def.h]: These functions aren't meant to be there always. Since i use this version as my daily driver, I'll need these things. And also, they'll make my job easier. If i remove them now, I'll have to switch between OG dwm and polywm just to check how it's going.

File: polygonwm.c (hunk 5 above)
Comment (Roidujeu) [reply polygonwm.c]: Okay. Will do.

File: polygonwm.c (hunk 6 above)
Comment (L04DB4L4NC3R) [reply polygonwm.c:2312]: Okay makes sense

File: config.def.h (hunk 4 above)
Comment (L04DB4L4NC3R) [reply config.def.h]: You can use a different user on a different tty to test it out

**Comment by Geek-ubaid** [comment]
@L04DB4L4NC3R PR is getting stale, see the changes requested and respond accordingly to the PR.

**Review by L04DB4L4NC3R — APPROVED** [review]

**Comment by L04DB4L4NC3R** [comment]
@Roidujeu add screenshots in the wiki and add a link to it in the README
